components/AccessWarning/AccessWarning.tsx

```diff
@@ -0,0 +1,27 @@
+import { Typography } from "@material-ui/core";
+import React from "react";
+
+type WarningCause = "iframe" | "token" | "invalid_token";
```

Comment:
I think this isn't perfect naming. "iframe" isn't a cause, its lack of iframe the problem. Or "no token", not "token"

So I suggest to make it explicit

```suggestion
type WarningCause = "not_in_iframe" | "missing_access_token" | "invalid_access_token";
```